### PR TITLE
qintelqsentry: New App - Initial Commit of the Qintel QSentry App

### PR DESCRIPTION
**Pull request by calvin-qintel**
#### Please check if your PR fulfills the following requirements:
- [X] Testing of all the changes has been performed (for bug fixes / features)
- [X] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [X] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [X] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [X] Verify all checks are passing.
- [X] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [X] Do NOT submit updates to dependencies unless it fixes an issue.
- [X] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [X] New App
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- initial release, providing the `ip reputation` action

## What is the current behavior? (OPTIONAL)
- NA

## What is the new behavior? (OPTIONAL)
- New App

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: qintelqsentry_connector.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        return phantom.APP_SUCCESS
+
+
+if __name__ == '__main__':
+
+    import sys
+
+    if (len(sys.argv) < 2):
+        print("No test json specified as input")
+        exit(0)
```
[the remark this answers is not in the document]
Comment (calvin-qintel) [reply qintelqsentry_connector.py]: done

File: qintelqsentry_connector.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        in_json = f.read()
+        in_json = json.loads(in_json)
+        print(json.dumps(in_json, indent=4))
+
+        connector = QSentryConnector()
+        connector.print_progress_message = True
+        ret_val = connector._handle_action(json.dumps(in_json), None)
+        print(json.dumps(json.loads(ret_val), indent=4))
+
+    exit(0)
```
[the remark this answers is not in the document]
Comment (calvin-qintel) [reply qintelqsentry_connector.py]: done

**Comment by haritas-crest** [comment]
Hi @calvin-qintel, could you please provide me the access rights to this repo(qintel:feat/qsentry_app)?

**Comment by haritas-crest** [comment]
The changes of this PR are incorporated into this- #2

**Pull request merged**